core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -0,0 +1,323 @@
+import * as d3 from "d3";
+
+import { Chart } from "../../chart";
+
+import "./horizontalBarChart.css";
+const initializingBarHeight = 2;
+const margin = {
+  top: 10,
+  right: 0,
+  bottom: 20,
+  left: 0,
+};
+const barValueMargin = 4;
+const barPadding = 0.2;
```

Comment:
From what I gather, when defining a single scale, the barPadding is simply used as a constant to create a padding between every bar that will be distributed evenly within the full range of the scale (`scale.padding(barPadding)`). Which ultimately has an effect on the size of the barWidth. In the case that the range is not the full size of the chart area (just like when grouping the data) I used barPadding as a proportion of the full space (full range of the scale) that is dedicated to padding i.e.:
`totalSpan * barPadding = paddingSpan`. Given that `totalSpan = paddingSpan + barSpan` i.e. the space the bars take up + the space the padding takes up, I solved for paddingSpan as to arrive to the calculation in the code expressed in terms of barSpan. 

Thinking about it now barPadding should probably be named paddingProportion and instead of letting  the scale `scale.padding(barPadding)` deciding the bar width, I think barWidth should a value it defaults to as to avoid massive bars when we have only a few data points.

Reply:
Please do these naming changes in verticalBarChart as well then